src/orange/detail/UnitIndexer.hh

```diff
@@ -26,44 +28,200 @@ class UnitIndexer
   public:
     //!@{
     //! \name Type aliases
-    using LocalSurface = std::tuple<UniverseId, SurfaceId>;
-    using LocalVolume  = std::tuple<UniverseId, VolumeId>;
-    using VecSize      = std::vector<size_type>;
+    using UnitIndexerDataRef
+        = UnitIndexerData<Ownership::const_reference, MemSpace::native>;
+    using DataRef
+        = Collection<size_type, Ownership::const_reference, MemSpace::native>;
+    using AllVals  = AllItems<size_type, MemSpace::native>;
+    using SizeId   = OpaqueId<size_type>;
+    using SpanIter = Span<const size_type>::const_iterator;
```

Comment:
`DataRef` is unused, and the rest of these should be private (since they're not part of the interface).